src/routes.ts

```diff
@@ -45,11 +45,11 @@ routes.post("/", async (req, res) => {
         log.debug(`Uploading bundle ${bundleId}`);
         await aws.upload({ bundle, bundleId, timestamp, referrer });
       } catch (e) {
-        log.debug(e);
+        log.debug("Error", e instanceof Error ? e.stack : e);
```

Comment:
This ensures tslog doesn't attempt to make a deep copy of the whole error object, which seems to be problematic for SDK v3 type errors.